stream/distributedlog/io/dlfs/src/test/java/org/apache/distributedlog/fs/TestDLFileSystem.java

```diff
@@ -61,7 +61,7 @@ public void testBasicIO() throws Exception {
         }
         assertTrue(fs.exists(path));
 
-        File tempFile = new File("/tmp/" + runtime.getMethodName());
+        File tempFile = new File(newTempDir(runtime.getMethodName()), "test.bin");
```

Comment:
this might actually cause the files never deleted. because new temp dir will generate a new directory and it is never deleted. 

I would suggest using JUnit temp folder.

https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html

https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieJournalForceTest.java#L72

Reply:
I tried to use the same style as DL tests, I am not still used to this codebase.

Temporary Folder is better for me. Will change